tools/clang/lib/SPIRV/SpirvBuilder.cpp

```diff
@@ -818,6 +818,15 @@ SpirvBuilder::createRayQueryOpsKHR(spv::Op opcode, QualType resultType,
   return inst;
 }
 
+SpirvInstruction *SpirvBuilder::createReadClock(SpirvInstruction *scope,
+                                                SourceLocation loc) {
+  assert(insertPoint && "null insert point");
```

Comment:
I am worried that other developers might be confused by the name `scope`.
I think if we check somehow it is an integer, it might be helpful.

Reply:
Done